templates/profile-edit.html

```diff
@@ -2,94 +2,80 @@
                                 <div class="col-md-6">
                                     <label class="small mb-1" for="inputOrgName">Occupation</label>
-                                    <input class="form-control" id="inputOrgName" type="text" placeholder="Occupation" value="{{ user.profession }}">
+                                    <input class="form-control" id="inputOrgName" type="text" name="profession" placeholder="Occupation" value="{{ form.profession.value }}">
                                 </div>
-                                <!-- Form Group (location)-->
                                 <div class="col-md-6">
-                                    <label class="small mb-1" for="inputLocation">Bio</label>
-                                    <input class="form-control" id="inputLocation" type="text" placeholder="Enter your Bio" value="{{ user.bio }}">
+                                    <label class="small mb-1" for="inputLocation">Status</label>
+                                    <input class="form-control" id="inputLocation" name="bio" type="text" placeholder="Enter your Bio" value="{{ form.bio.value }}">
                                 </div>
                             </div>
-                            <!-- Form Group (email address)-->
-                            <label class="small mb-1" for="inputEmailAddress">Email address</label>
-                            <div class="mb-3">
-                                <div class="form-outline mb-4 input-group flex-nowrap">
-                                    <span class="input-group-text fa-regular fa-envelope fs-5" id="addon-wrapping"></span>
-                                    <input type="email" class="form-control" id="inputEmailAddress" placeholder="Enter your email addres" aria-label="Firstname" aria-describedby="addon-wrapping" value="{{ user.email }}">
-                                </div>
-                            </div>
-                            <!-- Save changes button-->
+
                             <div class="">
                                 <a href="{% url 'profile' user.id %}" class="btn btn-primary mx-1">
                                     Profile
                                 </a>
-                                <button class="btn btn-primary" type="button">Save changes</button>
+                                <button class="btn btn-primary" type="submit">Save changes</button>
                             </div>
-                        </form>
+                        </div>
                     </div>
                 </div>
             </div>
-        </div>
+        </form>
     </div>
     <script>
         document.addEventListener("DOMContentLoaded",() => {
-            document.getElementById('imagefile').addEventListener('change', (event) => {
-                image = document.getElementById('imagefile')
+            document.getElementById('profileimagefile').addEventListener('change', (event) => {
```

Comment:
what about if we follow name conventions as "xxx-yyyy"
as example : "profile-image", "image-file"

Reply:
changed to profile-image and profile-file